app/client/components/consent/PrivacySettings.tsx

```diff
@@ -82,17 +87,75 @@ const buttonStyles = css`
   margin: 0 6px;
 `;
 
+const integStyles = css`
+  margin-right: 5px;
+  border: none;
+  border-radius: 15px;
```

Comment:
I know these styles are very likely to be pulled out, but they are half making an anchor look like a button - I say half as the anchor still has a visited link colour and an underline even though it appears to be a button - plus the border-radius is different to the actual buttons - it's not a blocker for me as these will 100% change but I want to raise my concern at introducing multiple differing button-like styles.